Index: libnfc-nci-2.0/src/nxp_nci_hal_libnfc-nci/src/gki/ulinux/gki_ulinux.cc
===================================================================
--- libnfc-nci-2.0.orig/src/nxp_nci_hal_libnfc-nci/src/gki/ulinux/gki_ulinux.cc
+++ libnfc-nci-2.0/src/nxp_nci_hal_libnfc-nci/src/gki/ulinux/gki_ulinux.cc
@@ -129,7 +129,11 @@ void GKI_init(void) {
   pthread_mutexattr_init(&attr);
 
 #ifndef __CYGWIN__
+#ifdef PTHREAD_MUTEX_RECURSIVE_NP
   pthread_mutexattr_settype(&attr, PTHREAD_MUTEX_RECURSIVE_NP);
+#else
+  pthread_mutexattr_settype(&attr, PTHREAD_MUTEX_RECURSIVE);
+#endif
 #endif
   p_os = &gki_cb.os;
   pthread_mutex_init(&p_os->GKI_mutex, &attr);
